giddy/ergodic.py

```diff
@@ -3,11 +3,12 @@
 """
 __author__ = "Sergio J. Rey <sjsrey@gmail.com>, Wei Kang <weikang9009@gmail.com>"
 
-__all__ = ["steady_state", "var_fmpt_ergodic", "fmpt"]
+__all__ = ["steady_state", "var_mfpt_ergodic", "mfpt"]
```

Comment:
Looks good! Thanks for changing the names. We will also need to change the [doc](https://pysal.org/giddy/api.html) accordingly.

Reply:
I think the doc's should change automatically since the source has changed?

Reply:
My understanding is we need to change the function name in the API .rst file https://github.com/pysal/giddy/blob/master/docsrc/api.rst

Reply:
good catch.